docs/contributor/01-00-installation.md

```diff
@@ -0,0 +1,165 @@
+  componentNameMapping: urlPath                                                               
+  type: ociRegistry
+```
+>**NOTE:** Because Pods inside the k3d cluster use the docker-internal port of the registry, it tries to resolve the registry against port 5000 instead of 5001. K3d has registry aliases but module-manager is not part of k3d and thus does not know how to properly alias `registry.localhost:5001`
+
+>**NOTE** Apply `"operator.kyma-project.io/use-local-template": "true"` to make sure that Lifecycle Manager will use the registry URL present in the ModuleTemplate.
+
+## Install modular Kyma on the k3d cluster
+
+1. Install the latest version `lifecycle-manager`.
+
+```bash
+kyma alpha deploy
+```
+If deployed successfully, you get the following output:
+
+```bash
+- Kustomize ready
+- Lifecycle Manager deployed
+- Module Manager deployed
+- Modules deployed
+- Kyma CR deployed
+- Kyma deployed successfully!
+
+Kyma is installed in version:
+Kyma installation took: 18 seconds
+
+Happy Kyma-ing! :)
+```
+
+Kyma installation is ready, but no module is activated yet.
+
+```bash
+kubectl get kymas.operator.kyma-project.io -A
+NAMESPACE    NAME           STATE   AGE
+kcp-system   default-kyma   Ready   71s
+```
+
+2. Apply `template.yaml` to register Istio as a module known for Lifecycle Manager.
+
+Istio Module is a known module, but it is not activated.
+
+```bash
+kubectl get moduletemplates.operator.kyma-project.io -A 
+NAMESPACE    NAME                  AGE
+kcp-system   moduletemplate-istio   2m24s
+```
+
+3. Give Lifecycle Manager permission to install CustomResourceDefinition (CRD) cluster-wide.
+
+>**NOTE:** This is a temporary workaround only required in the single-cluster mode
+
+Module-manager must be able to apply CRDs to install modules. In the remote mode (with control-plane managing remote clusters) it gets an administrative kubeconfig, targeting the remote cluster to do so. But in local mode (single-cluster mode), it uses Service Account and does not have permission to create CRDs by default.
+
+Run the following to make sure the module manager's Service Account is granted an administrative role:
+
+```bash
+kubectl edit clusterrole lifecycle-manager-manager-role
+```
+
+add
+
+```yaml
+- apiGroups:                                                                                                                  
+  - "*"                                                                                                                       
+  resources:                                                                                                               
+  - "*"                                                                                                                       
+  verbs:                                                                                                                      
+  - "*"
+```
+
+1. Enable Istio in Kyma
+
+```bash
+kyma alpha enable module istio
```

Comment:
I had to run: `kyma alpha enable module istio -c alpha`
because:
`Error: failed to validate module channel: the channel [regular] does not exist for the module template [istio]. choose from one of the available channels: [alpha]`